### PR TITLE
Trunk 5331

### PR DESCRIPTION
**Pull request by georgejunior**
I removed a comment

## Issue I worked on
https://issues.openmrs.org/browse/TRUNK-5331?filter=-1

### CODE REVIEW

**Comment by dkayiwa** [comment]
Did you take a look at this? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

File: api/src/main/java/org/openmrs/CohortMembership.java [hunk 1]
```diff
@@ -142,6 +141,6 @@ public int compareTo(CohortMembership o) {
 		if (ret == 0) {
 			ret = this.getUuid().compareTo(o.getUuid());
 		}
-		return ret;
+		return set;
```
Comment (samuelmale) [inline api/src/main/java/org/openmrs/CohortMembership.java]: Thanks for taking time to work on this, just a few comments;
1: Did you see the travis failure?
2: Are you even sure this file compiles?
3: Is this all the ticket required?

**Comment by georgejunior** [comment]
> Did you take a look at this? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

I followed it but the travis failed

File: api/src/main/java/org/openmrs/CohortMembership.java (hunk 1 above)
Comment (georgejunior) [reply api/src/main/java/org/openmrs/CohortMembership.java]: > Thanks for taking time to work on this, just a few comments;
> 1: Did you see the travis failure?
> 2: Are you even sure this file compiles?
> 3: Is this all the ticket required?

i saw the travis failure

**Comment by dkayiwa** [comment]
Are you sure that you followed all the instructions there?

File: api/src/main/java/org/openmrs/CohortMembership.java (hunk 1 above)
Comment (samuelmale) [reply api/src/main/java/org/openmrs/CohortMembership.java]: Can you fix it?

Comment (georgejunior) [reply api/src/main/java/org/openmrs/CohortMembership.java]: > Are you sure that you followed all the instructions there?

is there anything that i could have missed?

Comment (dkayiwa) [reply api/src/main/java/org/openmrs/CohortMembership.java]: Can you take a second look at you confirm that you did all that is mentioned on that pull request tips page?

**Pull request closed without merging**